### PR TITLE
SWSD Connector documentation for GA

### PR DESCRIPTION
**Pull request by LiranRomSamanage**
This is SWSD Connector documentation for GA
should be release - on Feb 12, 2024
Please make sure this documentation will publish when our connector version 1.1.1 will be published also

Thanks

### CODE REVIEW

**Comment by Court72** [comment]
@DougKlopfenstein

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by giladsmg — APPROVED** [review]
Looks good to me, thanks

**Comment by DougKlopfenstein** [comment]
Hi @LiranRomSamanage 
Please make the following changes so this pull request can be merged in a timely manner:
* Insure that the warnings listed above are fixed. Since all of the warnings are about links to Power BI documentation, just change the beginning of the link from "https://learn.microsoft.com/en-us/power-bi/connect-data..." to "/power-bi/connect-data..." (that is, remove "https://learn.microsoft.com/en-us" from the beginning of the links).
* Fix any conflicts between MicrosoftDocs/main and your fork/branch. Some of these conflicts might be caused by the name changes recently made in the overall Microsoft documentation. That is, your article might contain the word "datasets", which is now called "Semantic models". Or it might contain "Azure Active Directory" (or it's abbreviations Azure AD and AAD), which is now called "Microsoft Entra ID".
* In the table of contents (toc.yml), remove the "(Beta)" tag from the name of your connector.
* In the list of all connectors (connectors/index.md), remove the "(Beta)" tag from the name of your connector.
* Since your connector is supported by both Power Query Desktop and Power Query Online, you should add a section about connection to Power Query Online (and differentiate between the section documenting how to connect to Power Query Desktop). For example, see the [Azure SQL database](https://learn.microsoft.com/power-query/connectors/azure-sql-database) connector article for a sample of a connector that is supported in both Power Query Desktop and Power Query Online (as recommended in [Example connector articles](https://learn.microsoft.com/power-query/providing-user-documentation#example-connector-articles) in the connector certification documentation).

Note also that some of the smaller changes that you've made in this pull request will have to be undone because they do not follow Microsoft style rules (such as using "click" when selecting an item, or removing contractions when Microsoft style rules require contractions to make the docs more informal). The documentation build engine will reject these changes, so they will be undone by our editors before the article is released. Just wanted to let you know so you won't be surprised.

**Comment by LiranRomSamanage** [comment]
@DougKlopfenstein 
I've sync, remove Beta, fix conflicts.
1. As for the link to "https://learn.microsoft.com/en-us" it is not our content, but link to microsoft how-to guides. we would like our customers will know how to use the features like: incremental-refresh-configure#create-parameters and incremental-refresh-configure#define-policy and incremental-refresh-configure#create-parameters

how would you recommend to add such link?

2. As for the support Power Cloud and Desktop. we are only Desktop Connector. was and will be on this version. nothing changed.
3. As for Microsoft style rules - let me know what to change and I will change it.

**Comment by LiranRomSamanage** [comment]
@Court72  @DougKlopfenstein  -  please review, after my last commit and comment, Thanks

**Comment by RinatGil** [comment]
@DougKlopfenstein @Court72  @peter  @Court72 Hey 
Any update regarding that? Thx

**Comment by RinatGil** [comment]
@DougKlopfenstein @Court72 @peter @Court72 Hey
Any update regarding that? Thx

**Comment by DougKlopfenstein** [comment]
Hi @LiranRomSamanage 
Regarding the http://learn.microsoft.com/en-us links - because the links in your article are actually built into our (Microsoft's) documentation, you don't need the "http://learn.microsoft.com/en-us" portions of the link. In fact, that's why you are getting the error messages. Our build engine is seeing the absolute link you are providing and is requesting that you to remove the "http://learn.microsoft.com/en-us" prefix so that the link becomes a relative link. If you keep the prefix in the article, it will still work, but will only jump to a US English version article. By removing the prefix, the links in your article will then jump to the appropriate article depending on which language the user is currently using.

Regarding being only a Desktop connector - Your connector is also available in Power BI service and Fabric (Dataflow Gen2), both of which are online services. If they are available in online services, you should have a section about how to connect using Power Query Online. However, if you want to get this article out sooner rather than later, that section could be added later after merging what you have already.

**Comment by LiranRomSamanage** [comment]
@DougKlopfenstein 
I don't mind about open the external link on english only.
Is this PR is approve as it is, with warning? and can be merge?

**Comment by LiranRomSamanage** [comment]
@DougKlopfenstein @peter @Court72
Please approve, the connector is already published and available on desktop.
we need this documentation to be available too
Thanks

**Comment by LiranRomSamanage** [comment]
@Jak-MS - no one reply or approve this PR. the connector is already live and available, but only the documentation is holding back. Can you please approve so I can merge it?

File: powerquery-docs/connectors/solarwinds-service-desk.md
Comment (Jak-MS) [inline powerquery-docs/connectors/solarwinds-service-desk.md:84]:
```suggestion
2. Open Manage Parameters > Create parameters. Create two new DateTime parameters named RangeEnd and RangeStart ,with one month set as the range between them. These new parameters reduce download time. See: [create-parameters](/power-bi/connect-data/incremental-refresh-configure#create-parameters) for instructions.
```

File: powerquery-docs/connectors/solarwinds-service-desk.md
Comment (Jak-MS) [inline powerquery-docs/connectors/solarwinds-service-desk.md:106]:
```suggestion
    b. Incrementally refresh data starting 1 Days before the refresh date. See: [define-policy](/power-bi/connect-data/incremental-refresh-configure#define-policy) for instructions.
```

File: powerquery-docs/connectors/solarwinds-service-desk.md
Comment (Jak-MS) [inline powerquery-docs/connectors/solarwinds-service-desk.md:124]:
```suggestion
For instructions on configuring a scheduled refresh see: [scheduled-refresh](/power-bi/connect-data/refresh-scheduled-refresh).
```

**Comment by Jak-MS** [comment]
@LiranRomSamanage - I've made the link corrections as is our policy, but you'll need to commit them.  Unfortunately, I can't merge this without a sign off from @DougKlopfenstein.

@DougKlopfenstein - nudge to take another look, after the link corrections have been committed.  (I'm not sure why I'm unable to commit the changes directly).

**Comment by LiranRomSamanage** [comment]
@Jak-MS thanks, I removed them.
I still waiting (too long) for approval from @DougKlopfenstein 
please assist with this

File: powerquery-docs/connectors/solarwinds-service-desk.md
Comment (DougKlopfenstein) [inline powerquery-docs/connectors/solarwinds-service-desk.md:41]: Change "...and then click **Connect** to..." to "...and then select **Connect** to..." to meet Microsoft style rules.

File: powerquery-docs/connectors/solarwinds-service-desk.md
Comment (DougKlopfenstein) [inline powerquery-docs/connectors/solarwinds-service-desk.md:49]: Change "After completion of the steps above, you..." to "After you complete the previous steps, you..." to meet Microsoft style rules.

**Comment by DougKlopfenstein** [comment]
@LiranRomSamanage 
Sorry, but for some reason I was not being notified when you added comments to this pull request.
You don't need me to approve the article - you just need to add a `#sign-off` comment to this pull request. That should start the process of having our editors look at the article, then merging the article once any editorial changes are made.

**Comment by LiranRomSamanage** [comment]
#sign-off

**Comment by DougKlopfenstein** [comment]
#sign-off

**Comment by giladsmg** [comment]
#sign-off

**Comment by DougKlopfenstein** [comment]
Hi @LiranRomSamanage and @giladsmg 
The sign-off I did changed the do-not-merge label to ready-to-merge. At this point, our editors will take over to ensure that the article meets minimum spelling, grammar, and Microsoft style rules. Once all of the minimum requirements are met, the changes will be merged.

**Comment by Jak-MS** [comment]
@LiranRomSamanage @giladsmg - I will merge this now... 

I can't edit the files in your branch (this is unusual), but if I could I would make the following changes. They are not critical but are advisable.  You can make them on a follow-up PR.

1. the **Incremental Refresh Setting** header should read **Incremental Refresh setting** . 
2. code blocks and paragraphs that come under a numbered step should be indented 3 spaces 
3. UI text and buttons should be bolded, and our style prefers the word 'select' to 'click'.
    (example: `1. Click Done.` should be `1. Select **Done**.`) 
   also make sure there are periods at the end of each step. :) 

Thanks!

**Pull request merged**
